scripts/setup-skills.sh

```
@@ -0,0 +1,199 @@
+
+  local synced=0
+  local skipped=0
+  for skill in "${SYNC_SKILLS[@]}"; do
+    if [[ -d "${SHIPSHITDEV_SKILLS}/${skill}" ]]; then
+      # Only sync if source is newer or dest doesn't exist
+      if [[ ! -d "${AGENTS_SKILLS}/${skill}" ]]; then
+        cp -r "${SHIPSHITDEV_SKILLS}/${skill}" "${AGENTS_SKILLS}/"
+        synced=$((synced + 1))
+      else
+        # Replace with latest
+        rm -rf "${AGENTS_SKILLS}/${skill}"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)